src/functions/copy-behavior.ts

```diff
@@ -32,28 +32,44 @@ export function parseCodeBlockCopy(metastring?: string): CopyBehavior {
 }
 
 function copyFullText(text: string): string {
+  const isSingleLine = !text.includes('\n');
+
   let buff = '';
+  let prevToken: string | undefined = undefined;
 
   while (text.length) {
```

Comment:
I don't see this covering the business logic:
 - string from first line if `$` or `>>`
 - break out of the look when they reach a line with neither.

We don't need special cases for 1 or 2 or 10 lines. Just to apply the business logic above.

Hence my comment to create 1 keyword called `cmd` that takes care of  1 or multiple lines in the same algorithm.

Reply:
This covers full text copies only, multiline subsequent commands are not covered yet.

This fixes scenarios where snippets like:

```yml
      "raw_fact_json":
        json-key: "$"
        value:
          type: "jsonb"
          required: true
```

Are copied. In these kind of snippets we dont want to remove whitespace because
the indentation would break.

The cases like:

```shell
$ fluvio cluster create
$ fluvio topic create hello-world
```

Would need the `cmd` keyword suggested here:

https://github.com/infinyon/fluvio-docs/issues/209#issuecomment-2302294565

Because we can safely trim space in these multilines.